File: AC/ABC-TDD/161/b.go
package main

import "fmt"

func main() {
	var n, m, cnt, sum int
	fmt.Scan(&n, &m)
	a := make([]int, n)
	for i := 0; i < n; i++ {
		fmt.Scan(&a[i])
		sum += a[i]
	}
	border := float64(sum) / float64(4*m)
	for i := 0; i < n; i++ {
		if float64(a[i]) >= border {
			cnt++
		}
	}

	if cnt >= m {
		fmt.Println("Yes")
	} else {
		fmt.Println("No")
	}
}

File: AC/ABC-TDD/161/b.py
n, m = map(int, input().split())
a = list(map(int, input().split()))
border = sum(a)/(4*m)
cnt = 0
for i in range(n):
    if a[i] >= border:
        cnt += 1
if cnt >= m:
    print("Yes")
else:
    print("No")
